### DISCUSSION TITLE
Detect Orientation for OCR and Mixed Orientations

### DISCUSSION THREAD

**Discussion by ebeck-iq** [pymupdf/PyMuPDF#4165] [Q&A]
I'm using 1.24.13 at the moment.  From discussion #3254 PyMuPDF should be able to handle the orientation issues for text capture, but OCR was not mentioned.  Is this auto rotation handled in the OCR process as well?  
Additionally I have some scanned PDF's with mixed rotations (user error) and the OCR process is not producing usable text except from the cover page (normally would not be present).  We're able to deal with it using PyTesseract in testing, but would prefer to stick with PyMuPDF if possible.

**Comment by JorjMcKie** [comment]
Share an example please that is handled correctly by pytesseract but not by the Tesseract embedded in PyMuPDF.

**Reply by ebeck-iq** [reply]
I invited you as an outside collaborator to this repo - I hope that work OK.  There are client details on the doc.  
[https://github.com/iqbo-dev/priv-share/blob/main/5star-43312291.pdf](https://github.com/iqbo-dev/priv-share/blob/main/5star-43312291.pdf)

**Reply by ebeck-iq** [reply]
I also added the resulting extracted text in the same repo

**Comment by ebeck-iq** [comment]
Good morning - were you able to access the document?
